src/components/fx/layout_defaults.js

```diff
@@ -17,6 +17,7 @@ module.exports = function supplyLayoutDefaults(layoutIn, layoutOut, fullData) {
     }
 
     coerce('dragmode');
+    coerce('selectdirection', 'any');
```

Comment:
since the attribute has `dflt: 'any'` already, you do not need to include it here.